src/otx/algorithms/common/adapters/mmcv/clsincr_mixin.py

```diff
@@ -0,0 +1,64 @@
+"""Class Incremental Learning configuration mixin."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from typing import List
+
+from mmcv.utils import ConfigDict
+
+from otx.algorithms.common.adapters.mmcv.utils.config_utils import update_or_add_custom_hook
+
+
+class IncrConfigurerMixin:
+    """Patch config to support incremental learning for object detection."""
+
+    org_model_classes: List = []
+    model_classes: List = []
+
+    def configure_task(self, cfg):
+        """Patch config to support incremental learning."""
+        super().configure_task(cfg)
+        if "task_adapt" in cfg and self.task_adapt_type == "mpa":
```

Comment:
Why don't you take the change to change the name "mpa"? :)

Reply:
I changed it to default_task_adapt to make easy to search. If you have any opinion for name, please notice me.